guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -36,18 +36,29 @@ endif::[]
 With {ProjectName}, you can perform both BIOS and UEFI based PXE provisioning.
 Both BIOS and UEFI interfaces work as interpreters between the computer's operating system and firmware, initializing the hardware components and starting the operating system at boot time.
 
-ifdef::satellite[]
-For information about supported workflows, see https://access.redhat.com/solutions/2674001[Supported architectures and provisioning scenarios].
-endif::[]
-
+.PXE loaders
 In {Project} provisioning, the PXE loader option defines the DHCP `filename` option to use during provisioning.
 For BIOS systems, use the *PXELinux BIOS* option to enable a provisioned node to download the `pxelinux.0` file over TFTP.
 For UEFI systems, use the *PXEGrub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *PXEGrub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
-ifndef::satellite[]
-Use SecureBoot options to enable a client to download the `shim.efi` bootstrap bootloader that then loads the signed `grubx64.efi`.
-Other PXE loaders like PXELinux UEFI, Grub2 ELF or iPXE Chain, require additional configuration. These workflows are not documented at the moment.
+
+{ProjectName} supports UEFI Secure Boot.
```

Comment:
I have seen "Secure Boot" used but the capitalization looks odd to me. 

"SecureBoot" also looks strange.

Reply:
Shoud always be "Secure Boot" AFAIK:

* https://en.wikipedia.org/wiki/UEFI#Secure_Boot
* https://access.redhat.com/articles/5254641
* https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/secure-boot-landing?view=windows-11